src/util/system.h

```diff
@@ -128,6 +128,17 @@ inline bool IsSwitchChar(char c)
 #endif
 }
 
+struct DefaultArgHints {
```

Comment:
In commit "util: Add DefaultArgHints struct" (f5e3b93076403997ce9a3e23253fdaa5514753c9)

- I think this struct should be moved to `init.h` and not be in system.h, since it's defining options for `SetupServerArgs` and `AppInit` function, not interacting with the `ArgsManager` class or other system functions. It's fine to include `init.h` in gui code and https://github.com/bitcoin-core/gui/pull/35 makes the call to `SetupServerArgs` there more direct
- To be a little more consistent with naming and explicit, would rename `DefaultArgHints` to `ServerArgsOptions`, and rename `printtoconsole` to `printtoconsole_default`, and rename `server` to `server_default`. I think `ServerArgsOptions::gui` is a better name than `DefaultArgHints::gui` because the `gui` value is used to show or hide help text, not to determine default argument values.

Reply:
@ryanofsky 
Thank you for your review! [Updated](https://github.com/bitcoin/bitcoin/pull/19471#issuecomment-673554481).